frontend/src/pages/Clusters/Filesystems.js

```diff
@@ -27,22 +27,27 @@ import {
 import EmptyState from '../../components/EmptyState';
 
 function StorageId({storage}){
-  const settingsKey = `${storage.StorageType}Settings`
-  const fs_id = getIn(storage, [settingsKey, 'FileSystemId'])
+  const settingsKey = `${storage.StorageType}Settings`;
+  const canMountFileSystem = ['Efs', 'FsxLustre'].includes(storage.StorageType);
+  const idKey = canMountFileSystem ? 'FileSystemId' : 'VolumeId';
+  const detailsFragment = canMountFileSystem ? '#file-system-details' : '#volume-details';
+  const id = getIn(storage, [settingsKey, idKey]);
   const defaultRegion = useState(['aws', 'region']);
   const region = useState(['app', 'selectedRegion']) || defaultRegion;
+  const versionMinor = useState(['app', 'version', 'minor']);
+  const fsxStorageTypes = (versionMinor && versionMinor >= 2) ? ['FsxLustre', 'FsxOntap', 'FsxOpenZfs'] : ['FsxLustre'];
```

Comment:
issue(non-blocking): This looks a bit hard to maintain, as we risk of having version checks everywhere in codebase. I don't have a simple solution, but we could probably create a map `version -> storage types`, just to make it a little bit more explicit

What do you think?

Reply:
I think that it's a good point in general, however in this specific case we are talking about FSx storage types, and I don't expect them to add support for new FSx storage types in the near future... we can always add a map later if this is the case. However, I am maybe in the Storage section of the wizard having a "common" map would be useful, in which case we could add it now, @mtfranchetto opinions on this?

Reply:
For the time being I'll leave it like this as in the roadmap it's planned to have a proper feature flags/experimental features support.